staramr/subcommand/Search.py

```diff
@@ -182,6 +189,11 @@ def run(self, args):
             settings.insert(2, ['start_time', start_time.strftime("%Y-%m-%d %H:%M:%S")])
             settings.insert(3, ['end_time', end_time.strftime("%Y-%m-%d %H:%M:%S")])
             settings.insert(4, ['total_minutes', time_difference_minutes])
+            if args.include_resistance_phenotypes:
+                arg_drug_table = ARGDrugTable()
+                settings.extend(arg_drug_table.get_resistance_table_info())
```

Comment:
Done in f730d9e7ba2ea134f238bbc3c050f5d1425b0e0b

Reply:
When specify a custom location for the databases directory with `--dir`, the `info.ini` isn't written to the custom directory. The ini file is present with the default location. I can still get the DB info with `$ staramr db info <custom db directory>`.

Reply:
Thanks for letting me know.  The `info.ini` file is only when you build a database for distribution (under `dist/`) since I don't want to include the `.git` directory in the Python package here, but still want to record the commit id.  Other database builds keep the `.git` directory so this info is still available.

Reply:
That makes perfect sense! Thanks for clarifying